demo_scenes/HandTrackingDemoScene.gd

```diff
@@ -18,5 +18,6 @@ func _process(_dt):
 
 
 func _ready():
-	pass # Replace with function body.
+	vr.initialize();
```

Comment:
the demo scenes are not intended to run on their own; there should be only one entry point for vr.initialize() because at some point this will need customization. But as mentioned in a comment from you I will add a check for this in the demo scenes to print a warning when vr was not initialized

Reply:
I misunderstood the initialize_vr value in vr.initialize() and thought it protected against double initializations.

It's useful to run the scene on its own when working on just the hand tracking.  changed back

Reply:
thanks; 
for quick testing I usually do load the scene I'm working on directly in the GameMain.gd here: https://github.com/NeoSpark314/godot_oculus_quest_toolkit/blob/ef76d87a294b71f477b87bdf217f7c5ebd776ac0/GameMain.gd#L32

There you also see an example why it is in general not a good idea to direclty start a scene and have the vr.initialize in there: the WebVR initialization (which for technical reasons could not be insite the vr.initialize() would not be called.